app/controllers/pages_controller.rb

```diff
@@ -0,0 +1,3 @@
+class PagesController < ApplicationController
+  def welcome; end
```

Comment:
Just thoughts out loud: I'm not sure but this line can be removed.
Have you tried?

Reply:
Yes, but I think I'll leave it, just to feel more at ease